### PR TITLE
Replace TravisCI with GitHub Workflows

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: .github/workflows/go.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      run: |
+        # temporary fix for https://github.com/actions/setup-go/issues/14
+        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: go build -v ./cmd/...
```
Comment (mrd0ll4r) [inline .github/workflows/go.yaml]: Does this use modules by default now?

**Comment by mrd0ll4r** [comment]
Can I see this in action somewhere? :)

File: .github/workflows/go.yaml (hunk 1 above)
Comment (jzelinskie) [reply .github/workflows/go.yaml]: Yep

**Comment by jzelinskie** [comment]
https://github.com/jzelinskie/chihaya/pull/2

**Pull request merged**